src/models/thelio-mira-r1.0/README.md

```diff
@@ -0,0 +1,81 @@
+        - [NVIDIA GeForce RTX 3060 Ti](https://www.nvidia.com/en-us/geforce/graphics-cards/30-series/rtx-3060-ti/)
+            - 1x HDMI 2.1, 3x DisplayPort 1.4a
+        - [NVIDIA GeForce RTX 3070](https://www.nvidia.com/en-us/geforce/graphics-cards/30-series/rtx-3070/)
+            - 1x HDMI 2.1, 3x DisplayPort 1.4a
+        - [NVIDIA GeForce RTX 3080](https://www.nvidia.com/en-us/geforce/graphics-cards/30-series/rtx-3080/)
+            - 1x HDMI 2.1, 3x DisplayPort 1.4a
+        - [NVIDIA Quadro RTX 4000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-4000/)
+            - 3x DisplayPort 1.4, 1x DisplayPort over USB-C (VirtualLink)
+        - [NVIDIA Quadro RTX 5000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-5000/)
+            - 4x DisplayPort 1.4, 1x DisplayPort over USB-C (VirtualLink)
+        - [NVIDIA Quadro RTX 6000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-6000/)
+            - 4x DisplayPort 1.4, 1x DisplayPort over USB-C (VirtualLink)
+        - [NVIDIA Quadro RTX 8000](https://www.nvidia.com/en-us/design-visualization/quadro/rtx-8000/)
+            - 4x DisplayPort 1.4, 1x DisplayPort over USB-C (VirtualLink)
+        - [AMD Radeon RX 580](https://www.amd.com/en/products/graphics/radeon-rx-580)
+            - 2x DisplayPort 1.4, 2x HDMI 2.0b, 1x DVI-D (single link)
+        - [AMD Radeon RX 5700](https://www.amd.com/en/products/graphics/amd-radeon-rx-5700)
+            - 3x DisplayPort 1.4, 1x HDMI 2.0b
+- Memory
+    - Up to 128GB (4x32GB) quad-channel ECC Unbuffered DDR4 DIMMs @ 3200 MHz
+    - Tested with the following RAM modules (may ship with other tested modules):
+        - [Kingston HX432C16FB3/8](https://www.kingston.com/dataSheets/HX432C16FB3_8.pdf)
+        - [Kingston HX432C16FB4/64](https://www.kingston.com/dataSheets/HX432C16FB3K4_64.pdf)
+        - [Kingston HX432C16FB3K4/128](https://www.kingston.com/dataSheets/HX432C16FB3K4_128.pdf)
+        - [Micron MTA9ASF2G72AZ-3G2B1](https://www.micron.com/products/dram-modules/udimm/part-catalog/mta9asf2g72az-3g2/mta9asf2g72az-3g2b1)
+        - [Micron MTA18ASF4G72AZ-3G2B1](https://www.micron.com/products/dram-modules/udimm/part-catalog/mta18asf4g72az-3g2/mta18asf4g72az-3g2b1)
+- Networking
+    - 1x 2.5-Gigabit Ethernet port (Realtek RTL8125-CG 2.5G LAN)
+    - 1x 1-Gigabit Ethernet port (Intel I211-AT, 1 x Gigabit LAN Controller)
+    - Wi-Fi 6 (Intel AX200)
+- Power
+    - Three power supply options:
+        - [EVGA 220-B5-0650 (650W)](https://www.evga.com/products/product.aspx?pn=220-B5-0650-V1)
+        - [EVGA 220-G5-0750 (750W)](https://www.evga.com/products/product.aspx?pn=220-G5-0750-X1)
+        - [EVGA 220-G5-1000 (1000W)](https://www.evga.com/products/product.aspx?pn=220-G5-1000-X1)
+    - C13 power cable
+- Sound
+    - 3.5mm line out, line in, microphone jacks
+    - Optical S/PDIF out
+    - Up to 7.1-channel audio output
+    - AMD Starship/Matisse HD Audio Controller
```

Comment:
The motherboard manual lists ROG SupremeFX S1220A for the codec/chipset. We have audio codecs/chipsets listed for Massive and Mega, I think we should try to carry that level of detail forward.

I'm seeing that the S1220A may be a Realtek ALC1150. Do you have any lspci/lsusb or alsainfo output from the Mira to confirm the best way to list this?

Reply:
I didn't save the output, and I just returned Mira to the office to be rebuilt with the new chassis revision, so I can't confirm that. I got the "Starship/Matisse" line from lspci, but I might have grabbed the wrong bit of info. Feel free to update if you think it needs it.